app/src/main/java/org/fossasia/susi/ai/chat/ChatActivity.kt

```diff
@@ -216,10 +217,11 @@ class ChatActivity : AppCompatActivity(), IChatView {
         })
 
         rv_chat_feed.addOnScrollListener(object : RecyclerView.OnScrollListener() {
-            override fun onScrolled(recyclerView: RecyclerView?, dx: Int, dy: Int) {
+            @SuppressLint("RestrictedApi")
+            override fun onScrolled(recyclerView: RecyclerView, dx: Int, dy: Int) {
                 super.onScrolled(recyclerView, dx, dy)
 
-                if (linearLayoutManager.findLastCompletelyVisibleItemPosition() < rv_chat_feed.adapter.itemCount - 5) {
+                if (linearLayoutManager.findLastCompletelyVisibleItemPosition() < rv_chat_feed.adapter!!.itemCount - 5) {
```

Comment:
Remove null forced call

Reply:
Only safe (?.) or non-null asserted (!!.) calls are allowed on a nullable receiver of type RecyclerView.Adapter<(raw) RecyclerView.ViewHolder!>?

Reply:
Yup, use safe ones